odra-build/src/lib.rs

```diff
@@ -62,6 +62,12 @@ fn flags() -> Vec<String> {
     let module = std::env::var("ODRA_MODULE").unwrap_or_else(|_| "".to_string());
     let msg = format!("cargo:rustc-cfg=odra_module=\"{}\"", module);
     flags.push(msg);
+
+    flags.push("cargo:rerun-if-env-changed=ODRA_BACKEND".to_string());
```

Comment:
Why is this needed? What does it solve?

Reply:
It allows creating code specific for a backend. For example if something is not implemented in odra, we can run runtime:: code directly.